src/box/space.h

```diff
@@ -407,6 +407,13 @@ space_name(const struct space *space)
 	return space->def->name;
 }
 
+/** Get space engine. */
+static inline const struct engine *
+space_engine(const struct space *space)
```

Comment:
There's no need in this trivial helper. One can write `space->engine` and it will actually take less place than `space_engine(space)`.